lib/index.js

```diff
@@ -462,7 +462,14 @@ var OfflinePlugin = (function () {
 
         if (typeof validatedPath !== 'string' || _this4.assets.indexOf(validatedPath) === -1) return;
 
-        var hash = _loaderUtils2['default'].getHashDigest(compilation.assets[key].source(), 'sha1');
+        var source = compilation.assets[key].source();
+        if (source instanceof ArrayBuffer) {
```

Comment:
Do you know of any other situation that could cause source to be an `ArrayBuffer`? (It shouldn't matter though, since the same logic should apply.)